### PR TITLE
Add gripper gazebo control

### PR DESCRIPTION
**Pull request by DevwratJoshi**

### CODE REVIEW

**Comment by felixvd** [comment]
Could you rebase this and resubmit the PR against the branch `o2ac-changes`?

File: robotiq_gazebo/config/gripper_controller.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+gazebo_gripper:
```
Comment (felixvd) [inline robotiq_gazebo/config/gripper_controller.yaml]: Will this work with multiple grippers in the same simulation? What needs to be namespaced or prefixed?

File: robotiq_gazebo/src/mimic_joint_plugin.cpp [hunk 2]
```diff
@@ -47,9 +47,13 @@ MimicJointPlugin::MimicJointPlugin()
 
 MimicJointPlugin::~MimicJointPlugin()
 {
-  event::Events::DisconnectWorldUpdateBegin(this->updateConnection);
-
+  #if GAZEBO_MAJOR_VERSION < 9
+ 	 gazebo::event::Events::DisconnectWorldUpdateBegin(this->updateConnection);
+  #endif
+  //event::Events::DisconnectWorldUpdateBegin(this->updateConnection);
```
Comment (felixvd) [inline robotiq_gazebo/src/mimic_joint_plugin.cpp]: Delete unneeded comments

**Review by felixvd — CHANGES_REQUESTED** [review]
LGTM, but please add a comment, readme or namechange to `test.py` to be clearer on what that is testing. It looks more like an example to me, anyway

File: robotiq_gazebo/config/gripper_controller.yaml (hunk 1 above)
Comment (DevwratJoshi) [reply robotiq_gazebo/config/gripper_controller.yaml]: I have added a controller in the newest commit and renamed them both with the prefix indicating the arms they are attached to. A separate controller is needed for each gripper.

**Comment by felixvd** [comment]
Resubmitted as #2

**Pull request closed without merging**